### DISCUSSION TITLE
Simulate CdTe solar cell performance

### DISCUSSION THREAD

**Discussion by A-Udeh** [pvlib/pvlib-python#2260] [General]
Hey everyone,

I'm Anayo, a master's student at the University of Louisiana at Lafayette. My research focuses on optimizing the efficiency of CdTe-based thin film solar cells. We're experimenting with varying the compositions of elements in the absorber layer (Cd, Se, and Te) and adjusting the thicknesses of other layers to observe their effects on the electrical parameters.

We've tried using SCAPS-1D, but we were only able to change the thicknesses of the different layers. Our next step is to conduct a DFT study of the absorber layer with the desired composition of Cd, Se, and Te. We'll then use the property values obtained to simulate the solar cell performance on SCAPS 1D. However, my professor wants us to simulate it first to determine its feasibility before proceeding with the synthesis of CdTe.

Can I use Pvlib to do that?

thanks

**Comment by kandersolar** [comment]
pvlib does not provide any functionality for simulating solar cells based on composition and layer thicknesses.  Perhaps solcore is closer to what you need: https://www.solcore.solar/

**Reply by A-Udeh** [reply]
Thank you, sir. 
Let me check that out.
